db/migrate/20190130183841_create_price_tables.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePriceTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :price_tables do |t|
+      t.integer :trainees
+      t.integer :price_point
+
```

Comment:
You set up the association in the DB but didn't set up the foreign key 
Sessions should have a foreign key price_table_id
You should create a migration for that

Reply:
Done